src/components/TeamSection/Team.tsx

```diff
@@ -2,27 +2,40 @@ import React from "react";
 import Profile from "./Profile";
 import * as S from "./Team.styles";
 import "./Team.styles.ts";
-import { type TeamMember } from "./TeamInformation";
-import { executiveTeam } from "./TeamInformation";
+import { type TeamMember, executiveTeam, projectManagers, alumniTeam } from "./TeamInformation";
 
 type TeamProps = {
   teamMembers: TeamMember[];
   desktopView: boolean;
-  isExec: boolean;
+  activeCategory: "executives" | "projectManagers" | "alumni";
 };
+
+const getBaseLength = (category: "executives" | "projectManagers" | "alumni") => {
```

Comment:
you should turn this into a type or an enum
 
type myType = "executives" | "projectManagers" | "alumni";

Reply:
Addressed!